managing_infrastructure_and_inventory/_topics/storage_support_notes_about_analyzing_from_rhevm_3.0.md

```diff
@@ -44,3 +44,20 @@ storage. NFS storage does not have these requirements.
 
     3.  Select the server that relates to this instance of the
         {{ site.data.product.title_short }} appliance.
+
+3.  If using Logical Volume Manager (LVM), you will need to manually activate your volume groups
+    and logical volumes before performing the SmartState Analysis.
+
+    1. Edit `/etc/lvm/lvm.conf` and add volume groups to activate as read-only:
```

Comment:
And then here we should indicate that it is the lvm.conf file on the appliance that has to be modified